src/main/resources/templates/module_description.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head th:replace="layout/header :: header(~{::title})">
+
+    <title>Gaia - Module description</title>
+
+</head>
+<body class="dashboard dashboard_2">
+<div class="full_container">
+    <div class="inner_container">
+
+        <div th:replace="~{layout/sidebar}"></div>
+
+        <!-- right content -->
+        <div id="content">
+
+            <div th:replace="~{layout/topbar}"></div>
+
+            <div class="midde_cont">
+                <div class="container-fluid">
+                    <div class="row column_title">
+                        <div class="col-md-12">
+                            <div class="page_title">
+                                <h2> Module </h2>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="row">
+                        <div class="col-md-12">
+                            <div class="white_shd full">
+
+                                <div id="app"></div>
+
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+        </div>
+    </div>
+</div>
+
+<template id="template">
+    <div class="block module_description">
+        <section class="block_head row">
+            <div class="col-md-8">
+                <div class="row">
+                    <div class="col-md-2">
+                        <img src="/images/aws.png" alt="aws"/>
```

Comment:
ok for now.
I beleive that we should, for the future, just have a look at the terraform provider configuration to decide which image to show (AWS/GCP/Azure/Alibaba etc...)

Reply:
Or maybe, we can use a module field, hard-coded for now, as you did for the various metadata fields?

Maybe a `mainProvider` field?